packages/logger/logger.ts

```
@@ -0,0 +1,108 @@
+      level,
+      message,
+      data,
+      error,
+      module: this.getCallerModule(),
+    };
+
+    const formatted = this.formatLog(context);
+
+    switch (level) {
+      case LogLevel.ERROR:
+        console.error(formatted, error || '');
```

ESLint

[error] Unexpected console statement.

(no-console)